app/javascript/components/volunteer/Profile.jsx

```diff
@@ -31,33 +31,62 @@ class Profile extends React.Component {
       return <div className="">Loading...</div>;
     }
     let profileImage = <span>No Image</span>;
-    if (this.props.profileImage_url) {
-      profileImage = <img src={this.props.profileImage_url} />;
+    if (this.props.profile_image_url) {
```

Comment:
profileUrl = this.props.profile_image_url ? this.props.profile_image_url : default

profileImage = <>

resume = this.props.resume_url: <a className="fl pa0 pv0 ph3 resume-button f7 lh-m" src={this.props.resume_url}>resume</a> : <a className="fl pa0 pv0 ph3 resume-button f7 lh-m">resume</a>